includes/payments/class-payments-query.php

```diff
@@ -680,4 +746,98 @@ public function gateway_filter() {
 
 	}
 
+
+	/**
+	 * Get sql query
+	 *
+	 * Note: Internal purpose only. We are developing on this fn.
+	 *
+	 * @since  1.8.18
+	 * @access public
+	 * @global $wpdb
+	 *
+	 * @return string
+	 */
+	private function get_sql() {
```

Comment:
@DevinWalker I added new fn to which will return you SQL, currently, i set it to `private` because it will take time to integrate this will core but currently it is working fine with group query.